TestApp/TestApp/CustomViews/AtomicComponents/Molecules/TwoButtonView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  TwoButtonView.swift
+//  ECCBWallet
+//
+//  Created by Jonathan Bott on 11/4/19.
+//  Copyright © 2019 Jon Bott. All rights reserved.
+//
+
+import Foundation
+import DesignSystem
+
+public class TwoButtonView: UIView {
+
+    private let style1: Button.Style
+    private let style2: Button.Style
+    private var closure1: VoidClosure = { }
+    private var closure2: VoidClosure = { }
+
+    public init(style1: Button.Style, style2: Button.Style, buttonWidth: CGFloat, padding: CGFloat) {
+        self.style1 = style1
+        self.style2 = style2
+        super.init(frame: .zero)
+        guard let buttonBrand = BrandingManager.brand as? ButtonBrand else { Assert("BrandingManager.brand does not conform to ButtonBrand"); return }
+        
+        addSubview(button1)
+        addSubview(button2)
+        
+        let size = Button.Size(width: .half, height: .large)
+        
+        let offset = buttonWidth/2 + padding/2
+        
+        button1.pin(to: self, [.top(.xxsmall), .centerX(-offset),  .width(asConstant: 164), buttonBrand.heightPin(for: size)])
+        button2.pin(to: self, [.top(.xxsmall), .centerX(offset), .width(asConstant: 164), buttonBrand.heightPin(for: size)])
+        // Potentially change this widthPin / heightPin setup
+    }
+
+    @available(*, unavailable)
+    required public init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    public func set(buttonTitle1: String, buttonTitle2: String, closure1:@escaping VoidClosure, closure2:@escaping VoidClosure) {
+        button1.setTitle(buttonTitle1, for: .normal)
+        button2.setTitle(buttonTitle2, for: .normal)
+        self.closure1 = closure1
+        self.closure2 = closure2
+    }
+
+    @objc private func tapped(_ sender: Button) {
+        print("sender.tag: \(sender.tag)")
+        if sender.tag == 1 {
```

Comment:
I've heard tags like 1 are bad... I use unique things like `0xbeef` and make sure they're not magic that way i'm sure it's the right one... but nits anyway